sanic_jwt/endpoints.py

```diff
@@ -0,0 +1,152 @@
+from . import exceptions
+from . import utils
+from sanic.views import HTTPMethodView
+from sanic.response import json
+from sanic.response import text
+
+
+response = None
+
+
+# @bp.listener('before_server_start')
+# async def setup_claims(app, *args, **kwargs):
+#     app.auth.setup_claims()
+
+class BaseEndpoint(HTTPMethodView):
+    def __init__(self, config, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.config = config
+
+
+class AuthenticateEndpoint(BaseEndpoint):
+    async def options(self, request, *args, **kwargs):
+        return text('', status=204)
+
+    async def post(self, request, *args, **kwargs):
+        config = self.config
+        user = await utils.call(
+            request.app.auth.authenticate, request, *args, **kwargs)
+
+        access_token, output = await response.get_access_token_output(request,
```

Comment:
Would you mind if I format the code so it wouldn't end up being so long?

Reply:
go for it. I think my editor is set to 100, not 80.

Reply:
In fact, [it's 79](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) :sweat_smile:

Reply:
Done (not everything I guess) in d6ce8458dccad3d1f17c5e1dbb3dc82c79cbb607